packages/shared/src/lib/strings.ts

```diff
@@ -8,3 +8,6 @@ export const capitalize = (value: string): string =>
   (value && value[0].toUpperCase() + value.slice(1)) || '';
 
 export const anchorDefaultRel = 'noopener noreferrer';
+
+export const checkStringEquality = (value1: string, value2: string): boolean =>
```

Comment:
@sshanzel you might know, did we not already have such function somewhere?

Reply:
Do we really need a function for this? Just use `===`

Reply:
Can't exactly remember, though using `===`, it won't work when the other one is on uppercase and the other is on lowercase.

Reply:
Yeah in those cases you just do `toLowerCase()` inline where you need it, with above we are hiding implementation without any obvious reasons right?

Reply:
I think it's just a personal preference in terms of "code cleanliness", there was a lot of it in a single file which felt it was all over the place.

Reply:
We were comparing two values, one is probably lowercase already, but can't exactly remember which one, so in order to avoid testing whether we need lowercasing the other or not, we just do it both here.

Reply:
The function name could include "lowercase" to reflect what it does since implementation is hidden.